### ISSUE TITLE
run example error: `cargo run --example tour`

### ISSUE THREAD

**Issue by FuGangqiang** [hecrj/iced#80]
system: archlinux

error backtrace:

```
$ RUST_BACKTRACE=1 cargo run --example tour
    Finished dev [unoptimized + debuginfo] target(s) in 0.11s
     Running `target/debug/examples/tour`
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: IllFormed', src/libcore/result.rs:1165:5
stack backtrace:
   0: backtrace::backtrace::libunwind::trace
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/backtrace/libunwind.rs:88
   1: backtrace::backtrace::trace_unsynchronized
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/backtrace/mod.rs:66
   2: std::sys_common::backtrace::_print_fmt
             at src/libstd/sys_common/backtrace.rs:77
   3: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
             at src/libstd/sys_common/backtrace.rs:61
   4: core::fmt::write
             at src/libcore/fmt/mod.rs:1028
   5: std::io::Write::write_fmt
             at src/libstd/io/mod.rs:1412
   6: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:65
   7: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:50
   8: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:188
   9: std::panicking::default_hook
             at src/libstd/panicking.rs:205
  10: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:464
  11: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:373
  12: rust_begin_unwind
             at src/libstd/panicking.rs:302
  13: core::panicking::panic_fmt
             at src/libcore/panicking.rs:139
  14: core::result::unwrap_failed
             at src/libcore/result.rs:1165
  15: core::result::Result<T,E>::unwrap
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libcore/result.rs:933
  16: wgpu_glyph::builder::GlyphBrushBuilder<()>::using_fonts_bytes::{{closure}}
             at /home/fu/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/wgpu_glyph-0.6.0/src/builder.rs:44
  17: core::iter::adapters::map_fold::{{closure}}
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libcore/iter/adapters/mod.rs:694
  18: core::iter::traits::iterator::Iterator::fold::ok::{{closure}}
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libcore/iter/traits/iterator.rs:1828
  19: core::iter::traits::iterator::Iterator::try_fold
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libcore/iter/traits/iterator.rs:1709
  20: core::iter::traits::iterator::Iterator::fold
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libcore/iter/traits/iterator.rs:1831
  21: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libcore/iter/adapters/mod.rs:727
  22: core::iter::traits::iterator::Iterator::for_each
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libcore/iter/traits/itera
tor.rs:631
  23: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/liballoc/vec.rs:2031
  24: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/liballoc/vec.rs:2014
  25: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/liballoc/vec.rs:1901
  26: core::iter::traits::iterator::Iterator::collect
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libcore/iter/traits/iterator.rs:1493
  27: wgpu_glyph::builder::GlyphBrushBuilder<()>::using_fonts_bytes
             at /home/fu/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/wgpu_glyph-0.6.0/src/builder.rs:41
  28: iced_wgpu::text::Pipeline::new
             at wgpu/src/text.rs:36
  29: iced_wgpu::renderer::Renderer::new
             at wgpu/src/renderer.rs:64
  30: <iced_wgpu::renderer::Renderer as iced_native::renderer::windowed::Windowed>::new
             at wgpu/src/renderer.rs:417
  31: iced_winit::application::Application::run
             at ./winit/src/application.rs:115
  32: iced::application::Application::run
             at ./src/application.rs:140
  33: iced::sandbox::Sandbox::run
             at ./src/sandbox.rs:128
  34: tour::main
             at examples/tour.rs:10
  35: std::rt::lang_start::{{closure}}
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libstd/rt.rs:61
  36: std::rt::lang_start_internal::{{closure}}
             at src/libstd/rt.rs:48
  37: std::panicking::try::do_call
             at src/libstd/panicking.rs:287
  38: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:79
  39: std::panicking::try
             at src/libstd/panicking.rs:265
  40: std::panic::catch_unwind
             at src/libstd/panic.rs:396
  41: std::rt::lang_start_internal
             at src/libstd/rt.rs:47
  42: std::rt::lang_start
             at /rustc/50f8aadd746ebc929a752e5ffb133936ee75c52f/src/libstd/rt.rs:61
  43: main
  44: __libc_start_main
  45: _start
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by hecrj** [comment]
Thanks for the report.

This is the same issue as #69. Could you try again with the fix in #74?

**Comment by FuGangqiang** [comment]
yes, the pull request fix it
